src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
+  apiBase: 'https://ngshop-27da4.firebaseio.com'
```

Comment:
remember that we will have to change this for later

Reply:
Got it